### ISSUE TITLE
Add sponsor name to committee profile page

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4057]
**What we're after:**
Want to add candidate information to leadership PAC profile pages.

**Completion criteria:**
- [x] followup with @JonellaCulmer to see if we have/need a separate ticket for the mockup.
- [ ] Sponsor name is added to committee profile pages

**Design:**
![screen shot 2018-10-11 at 8 03 36 am](https://user-images.githubusercontent.com/31663028/46813946-af29f480-cd45-11e8-99b5-a63643d33b18.png)

**Comment by jason-upchurch** [comment]
blocked by #4612 (openfec) cc @jason-upchurch

**Comment by hcaofec** [comment]
An example api call:

https://api.open.fec.gov/v1/committee/C00432104/history/2020/?sort_hide_null=false&election_full=true&sort=-cycle&api_key=DEMO_KEY&sort_nulls_last=false&sort_null_only=false&page=1&per_page=20

**Comment by rfultz** [comment]
The PR for this has been merged so I'm closing the ticket. Not sure why it didn't happen automatically
